eng/automation/generate_data.py

```diff
@@ -71,12 +71,12 @@ def sdk_automation_typespec_project(tsp_project: str, config: dict) -> dict:
                 module,
             )
         else:
-            # check whether this is migration from Swagger
-            clean_sdk_folder_succeeded = clean_sdk_folder_if_swagger(sdk_root, sdk_folder)
+            # fallback to generate from a clean folder
```

Comment:
L45-51 and L74-79 are duplicate now.  Can this be refactored into a helper function to reduce code duplication?

Reply:
Thanks for the feedback.

Yes, it is duplicate a few lines. I tried to extract to e.g. `generate_typespec_project_in_clean_folder`, but find it hard to incorporate the condition (the func would return `clean_sdk_folder_succeeded=false` if failed to clean up, or current tuple if it can clean up). Feels code be less intuitive when extracted.
So keep the code as it is.